### PR TITLE
fix(AreaChart): Add dotted line area for null values

### PR DESCRIPTION
**Pull request by franckgaudin**

### CODE REVIEW

**Review by ofrogon — DISMISSED** [review]

**Review by vicky-comeau — DISMISSED** [review]

File: packages/AreaChart/src/AreaChart.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
   XAxis,
   YAxis,
   Tooltip,
+  Line,
```
Comment (ofrogon) [inline packages/AreaChart/src/AreaChart.tsx:10]: `Line` is never used

File: packages/AreaChart/src/helper/unavailableData.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import type { DataSet } from '../AreaChart';
+
+type DataWithRender = DataSet & {
+  render?: {
+    [index: string]: number | null;
+    uiScoreBackground: number | null;
+  };
+};
+
+export function getUniqueKeys(data: DataWithRender[]) {
```
Comment (ofrogon) [inline packages/AreaChart/src/helper/unavailableData.ts:10]: It is missing the method return type.

File: packages/AreaChart/src/helper/unavailableData.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  options: {
+    [index: string]: null | number;
+    uiScoreBackground: null | number;
+  };
+}
+
+export function getFakeScore(
+  data: DataWithRender[],
+  sequenceRanges: number[][]
+) {
```
Comment (ofrogon) [inline packages/AreaChart/src/helper/unavailableData.ts:59]: It is also missing the return type here

File: packages/AreaChart/src/helper/unavailableData.ts [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  }
+
+  if (start !== null) {
+    sequenceRanges.push([start, data.length - 1]);
+  }
+
+  return sequenceRanges;
+}
+
+interface Options {
```
Comment (ofrogon) [inline packages/AreaChart/src/helper/unavailableData.ts:49]: `Options` seems never used

**Review by vicky-comeau — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**